### PR TITLE
{box2d,liblangtag}: enable on darwin

### PR DESCRIPTION
**Pull request by veprbl**
###### Motivation for this change

These are dependencies of libreoffice (haven't figured how to fix that one yet).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 126643` at 586c254c run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>odpdown</li>
  </ul>
</details>
<details>
  <summary>6 packages skipped due to time constraints:</summary>
  <ul>
    <li>libreoffice (libreoffice-still)</li>
    <li>libreoffice-fresh</li>
    <li>libreoffice-fresh-unwrapped</li>
    <li>libreoffice-qt</li>
    <li>libreoffice-unwrapped (libreoffice-still-unwrapped)</li>
    <li>unoconv</li>
  </ul>
</details>
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>glabels</li>
    <li>libe-book</li>
    <li>liblangtag</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `fetchFromBitbucket`.
Near pkgs/development/libraries/liblangtag/default.nix:1:26:
```
  |
1 | { lib, stdenv, fetchurl, fetchFromBitbucket, autoreconfHook, gtk-doc, gettext
  |                          ^
```
</li>
  </ul>
</details>

File: pkgs/development/libraries/liblangtag/default.nix [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
   version = "0.6.3";
 
-  src = fetchFromBitbucket {
-    owner = "tagoh";
-    repo = pname;
-    rev = version;
-    sha256 = "10rycs8xrxzf9frzalv3qx8cs1jcildhrr4imzxdmr9f4l585z96";
+  # Artifact tarball contains lt-localealias.h needed for darwin
+  src = fetchurl {
+    url = "https://bitbucket.org/tagoh/liblangtag/downloads/liblangtag-0.6.3.tar.bz2";
```
Comment (7c6f434c) [inline pkgs/development/libraries/liblangtag/default.nix:11]: Maybe use variable expansion to make sure version is in sync?

Comment (veprbl) [reply pkgs/development/libraries/liblangtag/default.nix]: Absolutely, yes

**Comment by risicle** [comment]
`nixpkgs-review` happy, macos 10.15 ... oh.

**Pull request merged**